mesh/mesh_initializer.py

```diff
@@ -0,0 +1,40 @@
+__doc__ = """ Mesh Initializer using Pyvista """
+
+import pyvista as pv
+import numpy as np
+
+
+class MeshInitialize():
+    """
+    This Mesh Initializer class uses pyvista to import mesh files in the
+    STL or OBJ file formats and initializes the necessary mesh information.
+
+    How to initialize a mesh?
+    -------------------------
+
+    mesh = MeshInitialize(r('<filepath>'))
+
+    PS: Please be sure to add .stl / .obj at the end of the filepath, if already present, ignore.
```

Comment:
Add this under Notes and not PS

Reply:
Okay, I will add this change in the next PR.